code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -53,15 +53,14 @@ makeInstDoc = text "How to Build the Program:" <> contSep <> commandLine <> cont
     bkQuote3 <> contSep <> text "make build" <> contSep <> bkQuote3
 
 -- | Helper for giving instuctions and Input and Output files.
-inOutFile :: String -> (Maybe String, Maybe String) -> Doc
-inOutFile _    (Nothing, Nothing) = empty
-inOutFile _    (Nothing, _      ) = empty
-inOutFile _    (_      , Nothing) = empty
-inOutFile name (Just inFile, Just outFile) = doubleSep <>
```

Comment:
I liked this better; what if we have an input file but no output file? I'm not sure if I like them being wrapped in a tuple, though....

Reply:
Also make sure to be giving commits descriptive messages; there is a lot more being done here than just "fixing generated text". 😅